src/cbmc/CMakeLists.txt

```diff
@@ -1,73 +1,73 @@
 # Library
 file(GLOB_RECURSE sources "*.cpp" "*.h")
 list(REMOVE_ITEM sources
-    ${CMAKE_CURRENT_SOURCE_DIR}/cbmc_main.cpp
-)
+        ${CMAKE_CURRENT_SOURCE_DIR}/cbmc_main.cpp
+        )
 add_library(cbmc-lib ${sources})
 
 generic_includes(cbmc-lib)
 
 target_link_libraries(cbmc-lib
-    analyses
-    ansi-c
-    assembler
-    big-int
-    cpp
-    goto-checker
-    goto-instrument-lib
-    goto-programs
-    goto-symex
-    json
-    json-symtab-language
-    langapi
-    linking
-    pointer-analysis
-    solvers
-    statement-list
-    util
-    xml
-)
+        analyses
+        ansi-c
+        assembler
+        big-int
+        cpp
+        goto-checker
+        goto-instrument-lib
+        goto-programs
+        goto-symex
+        json
+        json-symtab-language
+        langapi
+        linking
+        pointer-analysis
+        solvers
+        statement-list
+        util
+        xml
+        )
 
 add_if_library(cbmc-lib bv_refinement)
 add_if_library(cbmc-lib jsil)
 
 # Executable
 add_executable(cbmc cbmc_main.cpp)
 if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
-# There is a cyclic dependency between analyses and ansi-c, which the
-# Makefile-based build system resolves by using --start-group, --end-group.
-# CMake lacks direct support (cf.
-# https://gitlab.kitware.com/cmake/cmake/-/issues/21511), so we ensure all
-# object files from libanalyses.a remain present.
-target_link_libraries(cbmc
-  cbmc-lib
-  -Wl,--whole-archive -Wl,${CMAKE_BINARY_DIR}/lib/libanalyses.a -Wl,--no-whole-archive
-)
+  # There is a cyclic dependency between analyses and ansi-c, which the
+  # Makefile-based build system resolves by using --start-group, --end-group.
+  # CMake lacks direct support (cf.
+  # https://gitlab.kitware.com/cmake/cmake/-/issues/21511), so we ensure all
+  # object files from libanalyses.a remain present.
+  target_link_libraries(cbmc
+          cbmc-lib
+          -Wl,--whole-archive -Wl,${CMAKE_BINARY_DIR}/lib/libanalyses.a -Wl,--no-whole-archive
+          )
 else()
-target_link_libraries(cbmc cbmc-lib)
+  target_link_libraries(cbmc cbmc-lib)
 endif()
 install(TARGETS cbmc DESTINATION ${CMAKE_INSTALL_BINDIR})
 
 # Man page
 if(NOT WIN32)
   install(
-    DIRECTORY ${CMAKE_SOURCE_DIR}/doc/man/
-    DESTINATION ${CMAKE_INSTALL_MANDIR}/man1
-    FILES_MATCHING PATTERN "cbmc*")
+          DIRECTORY ${CMAKE_SOURCE_DIR}/doc/man/
+          DESTINATION ${CMAKE_INSTALL_MANDIR}/man1
+          FILES_MATCHING PATTERN "cbmc*")
 endif()
 
 # bash completion
 if(NOT WIN32)
-  add_custom_command(OUTPUT "${CMAKE_SOURCE_DIR}/scripts/bash-autocomplete/cbmc.sh"
-    COMMAND "${CMAKE_SOURCE_DIR}/scripts/bash-autocomplete/extract_switches.sh"
-    DEPENDS $<TARGET_FILE:cbmc>
-  )
+  add_custom_command(OUTPUT "${CBMC_ROOT_DIR}/scripts/bash-autocomplete/cbmc.sh"
```

Comment:
This is actually one of the 4 lines that should have been modified, but still without the extra padding.

Reply:
fixed